utils/wasm-gen/src/utils.rs

```diff
@@ -0,0 +1,212 @@
+// This file is part of Gear.
+
+// Copyright (C) 2022 Gear Technologies Inc.
```

Comment:
Please turn the calendar over.
```suggestion
// Copyright (C) 2023 Gear Technologies Inc.
```